app/main.py

```diff
@@ -1,3 +1,83 @@
-def shop_trip():
-    # write your code here
-    pass
+from math import sqrt
+
+from app.car import FUEL_PRICE
+from app.customer import customers_
+from app.shop import shops_
+
+
+def shop_trip() -> None:
+    customers = customers_
+    shops = shops_
+    fuel_price = FUEL_PRICE
+
+    for customer in customers:
+        cheapest_shop = []
```

Comment:
Fix variable name. You store every shop info here, not just one cheapest_shop, as your variable name says